artemis-cli/src/test/resources/broker1.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version='1.0'?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can you rename this file as FileBrokerTest-broker1.xml and FileBrkerTest-broker2.xml?

It would be easier to find where that file belong to at a later point?